### PR TITLE
Removing unnecessary flags from tx pull

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
Superseding #3987.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 1f435ce053533dd15b4810740a3feeae11195a9d: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3989-0799e35f6f9afdd144b6276dc1855c75465d1413-x86-android.apk)
</details>

**Comment by why-not-try-calmer** [comment]
This is a bit of a mistery for me:
- lupdate is apparently present on the runner and accessible to the `update-translations.sh` script launched from the `translations.yml` workflow (causing the [error](https://github.com/opengisch/QField/actions/runs/4307021470/jobs/7511574663)); however
- lupdate is either absent or inaccessible to the `pull_translations.sh` script launched from the `sync-translations.yml` workflow.

**Comment by nirvn** [comment]
I think this is ready now, merging.

File: .github/workflows/sync-translations.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ on:
   schedule:
     - cron: "5 1 * * *"
   workflow_dispatch:
+  pull_request:
```
Comment (m-kuhn) [inline .github/workflows/sync-translations.yml:6]: @nirvn I think we want to kill that again, it was debug only

**Pull request merged**